Module 1: Capstone Project

PraisesPJMT opened the pull request:
## General Description
This capstone project is based on an online website for a [conference](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). <br/>
Features of the based website were personalised so that it has something unique. The project is based on a technology summit organised by the local state authority in Nigeria. The tech summit is called Kada Tech Summit 2022.

## Feature Description
- The project has two (2) pages: the home page and the about page.
- Each of these pages has versions for 2 different screen sizes:
  - Mobile: up to 768px wide.
  - Desktop: 768px or wider.
- The section "Featured speakers" was created dynamically in JavaScript.

## Project Presentation
The project presentation was made using loom with the following features:
- A description of the project.
- A demo of the project features.
- Highlight the impressive use of javascript to dynamically show event speakers.

Porject Presentation: [Video](https://www.loom.com/share/d17cd40a04b54aacaec8eea7c504e194)

## Project Deployment
The "capstone-deployment" branch on the GitHub repository was used as the live deployment page of the project.<br/>
Here is the link to the live page: [Kada Tech Summit 2022](https://praisespjmt.github.io/Kada-Tech-Summit/)

## This Feature was Tested in the Following Browsers
- [X] Chrome
- [X] Firefox
- [X] Edge
- [ ] Internet Explorer
- [ ] Safari

PraisesPJMT committed:
Add mobile Home page

PraisesPJMT committed:
Add mobile About page

PraisesPJMT committed:
Addapt home & about pages for desktop

PraisesPJMT committed:
Fix linter errors

PraisesPJMT committed:
Add past summit to about page

PraisesPJMT committed:
Add mobile menu open & close feature

PraisesPJMT committed:
Add mobile menu open bug

PraisesPJMT committed:
Add mobile menu open bug

PraisesPJMT committed:
Add dynamic speaker input on main.js file

PraisesPJMT committed:
Fix linter errors

PraisesPJMT committed:
Change feature speaker order

PraisesPJMT committed:
Add transition for header links

PraisesPJMT committed:
Add transition for main programs

PraisesPJMT committed:
Add transition for feature speakers

PraisesPJMT committed:
Add slide transition for header elements

PraisesPJMT committed:
Add fade animations for headline section

PraisesPJMT committed:
Add fade animations for about page headline section

PraisesPJMT committed:
Add animations to about page

PraisesPJMT committed:
Fix linter errors

PraisesPJMT committed:
Change some animation feature

Nemwel-Boniface commented on index.html:
```diff
@@ -0,0 +1,171 @@
+<body>
+    <header class="flex">
+        <div class="header-wrapper">
+            <div class="header-logo-container" data-aos="slide-down">
+                <a href="index.html" id="header-log" class="header-logo flex">
+                    <img class="h-logo-icon" src="images/software.png" alt="Kada Tech Logo"/>
+                    <div class="h-logo-name-container">
+                        <p class="h-logo-name secondary-color">Kada Tech <br/>Summit</p>
+                    </div>
+                </a>
+            </div>
+            <div class="mobile-control">
+                <i class="fa-solid fa-bars mobile-menu-control open" id="open"></i>
+                <i class="fa-solid fa-xmark mobile-menu-control close" id="close"></i>
+            </div>
```
You have done an amazing job with your mobile menu design :clap: . However, kindly note that according to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md), you were supposed to stick as much as possible to the provided [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). This said, there are a few corrections I would like to point out for you.

- [x] Kindly note that according to the Behance design on mobile view, the navbar is supposed to be positioned to the left side of the page and not on the right side of the page as it is in your design. Kindly fix this.
- [x] While you have made a great effort to also include a logo on the left side, kindly note that this is not supposed to be included here as per the project requirements. Kindly remove this section.

Below I will include screenshots of your design and the desired Behance design for the above-mentioned requested changes:.

**Your design:**

![image](https://user-images.githubusercontent.com/86318284/181538160-0e0ee8f7-d1c7-4e5d-9f5a-badea7719096.png)

**Desired Behance design:**

![image](https://user-images.githubusercontent.com/86318284/181539269-ecbdea5d-4587-41e7-8cd3-0408d3fc1ded.png)

**Kindly make this change to the mobile menu for the about page as well**

PraisesPJMT replied:
Thanks for the observation! I just made the changes as recommended!

Nemwel-Boniface commented on index.html:
```diff
@@ -0,0 +1,171 @@
+                <i class="fa-solid fa-bars mobile-menu-control open" id="open"></i>
+                <i class="fa-solid fa-xmark mobile-menu-control close" id="close"></i>
+            </div>
+            <nav data-aos="slide-down">
+                <a href="about.html" id="nav-about">About</a>
+                <a href="#" id="nav-program">Program</a>
+                <a href="#" id="nav-join">Join</a>
+                <a href="#" id="nav-sponsor">Sponsor</a>
+                <a href="#" id="nav-news">News</a>
+                <span id="license">CC Campaign</span>
+            </nav>
```
- [x] You have done a great job styling your navigation bar. However, I have realized that the top bar which houses the social media links is not present in your design. Kindly include it to make your design match the Behance design.

**This is how your design currently looks like**

![image](https://user-images.githubusercontent.com/86318284/181546374-d9c0f290-7027-486e-bf5d-dad24cde2a21.png)


**This is the desired  Behance design:**

![image](https://user-images.githubusercontent.com/86318284/181313006-3dc9f519-78ff-4847-be7f-0603ccdb8948.png)

**Kindly include it as well in your about page as I have realised that it is missing in that page as well**

PraisesPJMT replied:
Thanks for the observation. I just added the upper navigation bar.

Nemwel-Boniface commented on index.html:
```diff
@@ -0,0 +1,171 @@
+                <a href="#" id="nav-join">Join</a>
+                <a href="#" id="nav-sponsor">Sponsor</a>
+                <a href="#" id="nav-news">News</a>
+                <span id="license">CC Campaign</span>
```
- [x] Great job styling your span, However, to meet all the project requirements, you are supposed to stick as much as possible to the Behance in fonts, colors, and typography. Kindly change the color of your span to match the Behance design.

**This is how your span currently looks like:**

![image](https://user-images.githubusercontent.com/86318284/181547206-65436f36-995b-4007-9839-8ec72b448056.png)

**This is the desired Behance design:**

![image](https://user-images.githubusercontent.com/86318284/181339815-6a659f18-62b8-4fc9-a734-677f203a0e5c.png)

PraisesPJMT replied:
Thanks for the observation! I just changed the color of the Behance in accordance with the design.

PraisesPJMT replied:
Thanks for the observation! I just changed the color of the Behance in accordance with the design.

Nemwel-Boniface commented on about.html:
```diff
@@ -0,0 +1,222 @@
+            Some Rights Reserved.</p>
+    </div>
+
+</footer>
```
- [x] Great job styling your about page footer. However, the desktop version according to the project requirements is supposed to have a dark background and not a white background as it is currently on your design. Kindly make this change.

**For reference, this is how the about page footer looks like in the Behance design:**

![image](https://user-images.githubusercontent.com/86318284/181560411-d014c0ab-f5f9-4706-b086-57ae895539e0.png)

PraisesPJMT replied:
Yes, I just realize this! Thanks for the observation! I just changed the background to a dark colour.

Nemwel-Boniface commented on index.html:
```diff
@@ -0,0 +1,171 @@
+                    </div>
+                </div>
+                </div>
+            <div class="button">
+                <a href="#" class="join-summit white">Join Kada Tech Summit 2022</a>
+            </div>
```
- [x] You have made a great effort In styling this link. However, In the desktop view, it is not supposed to look like a button but like a link. You can achieve this by setting border, outline to none and removing the background color in the desktop view. Kindly make this change.

**This is how your section currently looks like:**

![image](https://user-images.githubusercontent.com/86318284/181564708-67d9a811-40fe-4373-86bc-a072bdd668f0.png)

**This is the desired Behance design:**

![image](https://user-images.githubusercontent.com/86318284/181564921-acd8f29f-530a-4a15-ab17-b82ae0e0a882.png)

PraisesPJMT replied:
Thanks so much for the observation. I just changed the background to none for the Desktop view.

Nemwel-Boniface commented on js/main.js:
```diff
@@ -0,0 +1,123 @@
+            + 'and CEO of AppsTech, a Bethesda, Maryland-based global provider'
+            + ' of enterprise application solutions. AppsTech, which was founded '
+            + 'in 1999 now has clients in more than 40 countries on 3 continents. ',
+  },
```
- [x] [OPTIONAL] - It would be a good thing to reduce this text so that on mobile view your users do not have a wrapped view of this section.

PraisesPJMT replied:
I just reduced the text contents of all speakers' descriptions to improve the mobile experience. Thanks for the observation.

Nemwel-Boniface commented on about.html:
```diff
@@ -0,0 +1,222 @@
+                        At DTS22, you’ll find all your tech needs under one roof. Learn how to
+                        work smarter, not harder. Sharpen your instincts to ensure you stay one
+                        step ahead of the competition. Expand your knowledge with insights from
+                        inspirational industry leaders and upskill with exposure to the latest
+                        cutting-edge technology.</p>
+                </div>
+            </div>
+            <!--Why KDS 02- Integrate-->
+            <div class="kts-wrapper" data-aos="slide-up" data-aos-delay="50" data-aos-duration="1000">
+                <div class="kts flex">
+                    <div class="kts-headings flex">
+                        <img src="images/why-kts-02.png"  alt="Integrate Icon" class="kts-icon">
+                        <h3 class="kts-title secondary-color">Integrate</h3>
+                    </div>
+                    <p class="kts-description bold">Unique opportunities to meet with
+                        C-Suite attendees and investors. Connect with responsive tech experts
+                        from all over the world. Share ideas with the best influencers in their
+                        fields and participate in thought-provoking panel sessions. Exchange
+                        knowledge and strategies with an energised global community.</p>
+                </div>
+            </div>
+            <!--Why KDS 02- Accelerate-->
+            <div class="kts-wrapper" data-aos="slide-up" data-aos-delay="50" data-aos-duration="1000">
+                <div class="kts flex">
+                    <div class="kts-headings flex">
+                        <img src="images/why-kts-03.png"  alt="Accelerate Icon" class="kts-icon">
+                        <h3 class="kts-title secondary-color">Accelerate</h3>
+                    </div>
+                    <p class="kts-description bold">Discover new ways to build your brand and
+                        get immediate traction. Amplify and scale your business for a bigger audience.
+                        Connect and grow your database with an engaged audience of over 10,000 informed
+                        delegate. Get on the investor track with unique access to the decision-makers
+                        at specially curated events. Step-up your engagement with global thought-leaders.</p>
+                </div>
+            </div>
+        </div>
+        <div class="button">
+            <a href="#" class="join-summit white">Join Kada Tech Summit 2022</a>
+        </div>
+    </section>
```
- [ ] While you have made a great effort in creating and styling this section, kindly note that it is not part of the project according to the provided Behance design and should therefore not be included in the project. Kindly remove this section and in its place put the design that is required as per the Behance design.

**This is the desired section. You are allowed to personalize it to your wish but not entirely alter the design**

![image](https://user-images.githubusercontent.com/86318284/181570585-aa7e2921-8c27-47f4-98be-2c5fe742b090.png)

PraisesPJMT replied:
Well Noted! I just removed both the Organizers section and the Event section. And I replaced them with a section about the event logo as contained in the design guideline.

Nemwel-Boniface reviewed (changes requested):
## Hello @PraisesPJMT           :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

You have done an excellent job in creating your capstone project. I really like the animations that you have included in the project. However, I have noticed that they may be the ones making your project load slowly in the live demo. 

All in all great job :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!

![](https://media.giphy.com/media/KyBesf8kNT6s88SEgb/giphy.gif)

## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  
Excellent project demo video ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions

- [ ] I have realized that you did not use the cocogoose font anywhere in your project. You can use this import statement in your CSS file ```@import URL('http://fonts.cdnfonts.com/css/cocogoose');``` and use it in your individual sections as follows ```font-family: 'COCOGOOSE', sans-serif;```. 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

PraisesPJMT committed:
Change mobile menu controls to the left

PraisesPJMT committed:
Add upper nav bar in heading section

PraisesPJMT committed:
Change color for Behance

PraisesPJMT committed:
Change footer background color

PraisesPJMT committed:
Change 'join event' button background to none for desktop

PraisesPJMT committed:
Reduce description of speakers for better mobile view experience

PraisesPJMT committed:
Replace organiser section with event logo section

PraisesPJMT committed:
Resize even logo for better mobile experience

PraisesPJMT commented:
## Feature Description
The following feature changes were made based on code review recommendations of the Pull Request.
- Changed mobile controls (open and close) to the left in accordance with the design guidelines
- Changed the colour of the 'CC Campaign' text in the header section.
- Changed the background color of the footer section to a darker color.
- Change 'join event' button background to none for desktop.
- Reduce description of speakers for better mobile view experience.
- Rmoved both the Organizers section and the Event section with a section about the event logo as contained in the design guideline.

## This Feature was Tested in the Following Browsers
- [X] Chrome
- [X] Firefox
- [X] Edge
- [ ] Internet Explorer
- [ ] Safari

AnselemOdims reviewed (approved):
### Hi  @PraisesPJMT 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

PraisesPJMT commented:
Thanks so much for the amazing review!!!

Happy coding guys!! :+1:
